docs/fabric/MAS_router/mas_next_tile_fifo_arb.md

```diff
@@ -42,10 +42,24 @@ Data Flow Description:
 3. It determines the priority of different cardinal directions based on the target address and the current tile's ID.
 4. The selected cardinal direction for each request is then provided as output.
```

Comment:
if i rememeber correctly, this is the block where the routing algorithm actually takes place.
Currently we are using a "x,y" aproch, setting the next tiles to reach the 'x' and then reach the 'y'
In future we may want to improve this by having a different routing scheme. (Example responses will readh 'y' and then 'x'?)

Reply:
done